.github/workflows/tools-api-docs.yml

```diff
@@ -0,0 +1,41 @@
+name: nf-core/tools API docs
+on:
+  push:
+    branches: [master, dev]
+
+jobs:
+  build-n-publish:
+    name: Build and publish nf-core to PyPI
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Check out source-code repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip Sphinx sphinxcontrib-napoleon
+          pip install .
+
+      - name: Build HTML docs
+        run: make --directory ./docs/api html
+
+      - name: Push docs to api-doc branch
+        if: github.repository == 'nf-core/tools'
```

Comment:
I did not knew we could do that

Reply:
Pretty nice, eh? Can do logic with pretty much any of the github variables here too - there are a lot!

Reply:
I'm pretty sure we can improve branch protection with that

Reply:
but that'll be for another release